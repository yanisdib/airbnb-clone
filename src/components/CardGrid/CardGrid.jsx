import styled from 'styled-components';


export default styled.div`
    display: grid;
    grid-template-columns: 382px 382px 382px 382px;
    row-gap: 24px;
    column-gap: 24px;
`;