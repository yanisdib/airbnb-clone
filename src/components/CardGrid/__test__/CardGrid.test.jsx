import { mount } from 'enzyme';

import CardGrid from '../CardGrid';


describe('CardGrid component', () => {
    let wrapper;

    beforeEach(() => wrapper = mount(<CardGrid />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})