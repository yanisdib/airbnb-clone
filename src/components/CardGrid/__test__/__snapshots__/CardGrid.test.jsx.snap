// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardGrid component renders correctly without crashing 1`] = `
<styled.div>
  <div
    className="sc-bdvvtL jbohyQ"
  />
</styled.div>
`;
